app/components/certificate/certificate-request.tsx

```diff
@@ -19,7 +19,7 @@ export default function CertificateRequestView({ domain }: CertificateRequestVie
       paddingX="1"
     >
       <Description
-        description="Initial: Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s"
+        description="By clicking Request a Certificate button, you will have an HTTPS certificate for all of your projects. "
```

Comment:
"By clicking the Request a Certificate button, you will initiate a request for an HTTPS certificate that you can use with any project connected to your DNS records."